test-framework/core/pom.xml

```diff
@@ -38,7 +38,7 @@
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-engine</artifactId>
+            <artifactId>junit-jupiter</artifactId>
```

Comment:
I am not sure if we plan to use junit jupiter params in the future, so this is more like a proposal